tests/freqai/test_freqai_interface.py

```diff
@@ -425,6 +432,7 @@ def test_plot_feature_importance(mocker, freqai_conf):
     freqai = strategy.freqai
     freqai.live = True
     freqai.dk = FreqaiDataKitchen(freqai_conf)
+    freqai.dk.live = True
```

Comment:
seems like this change (or one in the actual files) causes the windows-test to fail as a mock object is used as path (causing an invalid path).
on linux/macos, that'll be fine as that accepts "almost everything" as valid path - but windows is a bit picky here.

Reply:
![image](https://user-images.githubusercontent.com/5024695/228647550-3877edd9-c902-40b5-8b4a-4fb00efae250.png)

clearly a test-issue ... 
the mock is set from load_data in line 510 from datadrawer  - where `dk.model_filename = self.pair_dict[coin]["model_filename"]` ... (and pair_dict is a magicmock, which is assigned right in that test.

Now it's clearly caused by the switch to "live = True" - otherwise this path is not called .... i'm not entirely sure what the actual fix is though, other than something VERY specific for this test - which i'm not sure is a good idea.

Reply:
Running that test on linux/mac also (somewhat) causes this - though it does not fail.

have a clean checkout ... it'll generate a new directory named `MagicMock` ... ;) 

![image](https://user-images.githubusercontent.com/5024695/228649484-5868f0fc-b7a6-4cf0-aaf6-284bc408a4d2.png)